bundles/org.eclipse.ui.workbench/Eclipse UI/org/eclipse/ui/internal/CloseAllHandler.java

```diff
@@ -83,4 +122,37 @@ public void collectExpressionInfo(ExpressionInfo info) {
 		}
 		return enabledWhen;
 	}
+
+	/**
+	 * <p>
```

Comment:
please remove "<p></p>" - html formatted javdoc on private code is a waste, people will only read the comment.